scripts/Jenkinsfile

```diff
@@ -137,6 +137,14 @@ pipeline {
                         }
                     }
                 }
```

Comment:
Please, add a e2e robot test, when model validation is failed.
Useful links:
* [Example of test trainer](https://github.com/odahu/odahu-flow/blob/develop/odahuFlow/robot/odahuflow/robot/training_data_helper.py#L61)
* [Usage of the test trainer](https://github.com/odahu/odahu-flow/blob/develop/odahuFlow/tests/e2e/robot/tests/training/traininig_data.robot)